src/assets/js/utils.js

```diff
@@ -95,7 +95,21 @@ export async function saveSelectedProgramsAnswer(answerId, selectedPrograms, tok
 export async function saveYesNoAnswer(answerId, justifyAnswer, shortAnswer, token) {
```

Comment:
You should change this function name to something more descriptive 'saveAgreementAnswer' for instance.